lib/ansible/modules/files/copy.py

```diff
@@ -38,9 +38,10 @@
     version_added: '1.1'
   dest:
     description:
-    - Remote absolute path where the file should be copied to.
+    - Remote path where the file should be copied to.
     - If I(src) is a directory, this must be a directory too.
     - If I(dest) is a non-existent path and if either I(dest) ends with "/" or I(src) is a directory, I(dest) is created.
+    - If I(dest) is a relative path, the starting directory is dedermined by the remote host.
```

Comment:
s/dedermined/determined/

Reply:
@bcoca Thanks for spotting that! I recently switched keymaps. I've amended the commit.

Reply:
fyi, its very very ironic i catch a misspelling ...